index.bs

```diff
@@ -736,6 +736,183 @@ Depending on the underlying platform, the user agent <span class=allow-2119>may<
+      {{MLGraphBuilder/pad()}},
+      {{MLGraphBuilder/reshape()}},
+      {{MLGraphBuilder/slice()}},
+      {{MLGraphBuilder/split()}},
+      {{MLGraphBuilder/transpose()}},
+      {{MLGraphBuilder/resample2d()}},
+      {{MLGraphBuilder/reverse()}},
+      {{MLGraphBuilder/tile()}}
+      </td>
+    </tr>
+    <tr>
+      <td>Tensor quantization</td>
+      <td>
+      {{MLGraphBuilder/quantizeLinear()}},
+      {{MLGraphBuilder/dequantizeLinear()}}
+      </td>
+    </tr>
+    <tr>
+      <td>Tensor casting</td>
+      <td>
+      {{MLGraphBuilder/cast()}}
+      </td>
+    </tr>
+    <tr>
+      <td>Mathematics</td>
+      <td>
+      {{MLGraphBuilder/add()}},
+      {{MLGraphBuilder/sub()}},
+      {{MLGraphBuilder/mul()}},
+      {{MLGraphBuilder/div()}},
+      {{MLGraphBuilder/max()}},
+      {{MLGraphBuilder/min()}},
+      {{MLGraphBuilder/pow()}},
+      {{MLGraphBuilder/abs()}},
+      {{MLGraphBuilder/ceil()}},
+      {{MLGraphBuilder/cos()}},
+      {{MLGraphBuilder/erf()}},
+      {{MLGraphBuilder/exp()}},
+      {{MLGraphBuilder/floor()}},
+      {{MLGraphBuilder/identity()}},
+      {{MLGraphBuilder/log()}},
+      {{MLGraphBuilder/neg()}},
+      {{MLGraphBuilder/reciprocal()}},
+      {{MLGraphBuilder/sin()}},
+      {{MLGraphBuilder/sqrt()}},
+      {{MLGraphBuilder/tan()}},
+      {{MLGraphBuilder/sign()}}
+      </td>
+    </tr>
+    <tr>
+      <td>Logical</td>
+      <td>
+      {{MLGraphBuilder/equal()}},
+      {{MLGraphBuilder/notEqual()}},
+      {{MLGraphBuilder/greater()}},
+      {{MLGraphBuilder/greaterOrEqual()}},
+      {{MLGraphBuilder/lesser()}},
+      {{MLGraphBuilder/lesserOrEqual()}},
+      {{MLGraphBuilder/logicalNot()}},
+      {{MLGraphBuilder/where()}},
+      {{MLGraphBuilder/logicalAnd()}},
+      {{MLGraphBuilder/logicalOr()}},
+      {{MLGraphBuilder/logicalXor()}}
+      </td>
+    </tr>
+    <tr>
+      <td>Convolution</td>
+      <td>
+      {{MLGraphBuilder/conv2d()}},
+      {{MLGraphBuilder/convTranspose2d()}}
+      </td>
+    </tr>
+    <tr>
+      <td>Matrix multiplication</td>
+      <td>
+      {{MLGraphBuilder/matmul()}},
+      {{MLGraphBuilder/gemm()}}
+      </td>
+    </tr>
+    <tr>
+      <td>Pooling</td>
+      <td>
+      {{MLGraphBuilder/averagePool2d()}},
+      {{MLGraphBuilder/l2Pool2d()}},
+      {{MLGraphBuilder/maxPool2d()}}
+      </td>
+    </tr>
+    <tr>
+      <td>Activation</td>
+      <td>
+      {{MLGraphBuilder/clamp()}},
+      {{MLGraphBuilder/elu()}},
+      {{MLGraphBuilder/gelu()}},
+      {{MLGraphBuilder/hardSigmoid()}},
+      {{MLGraphBuilder/hardSwish()}},
+      {{MLGraphBuilder/leakyRelu()}},
+      {{MLGraphBuilder/linear()}},
+      {{MLGraphBuilder/prelu()}},
+      {{MLGraphBuilder/relu()}},
+      {{MLGraphBuilder/sigmoid()}},
+      {{MLGraphBuilder/softmax()}},
+      {{MLGraphBuilder/softplus()}},
+      {{MLGraphBuilder/softsign()}},
+      {{MLGraphBuilder/tanh()}},
+      {{MLGraphBuilder/triangular()}}
```

Comment:
The triangular matrix masking function doesn't feel like [activation](https://en.wikipedia.org/wiki/Activation_function), but I don't know what category it fits in either 🤔. Are there cases where trilu acts as activation that I'm unaware of?

Reply:
> Does it make sense to add a new "Tensor masking"
> Also "tensor rearrangement" op? 

Content with either.